### DISCUSSION TITLE
epinowcast organisation

### DISCUSSION THREAD

**Discussion by seabbs** [epinowcast/epinowcast#161] [Polls]
Currently, the package lives within the `epiforecast` Github organisation. This is mostly motivated by it being the research group in which I work vs any principled reason. Some of the downsides to this are that the organisation has a different focus to our project, and that this potentially limits our ability to develop additional helper packages (or other forms of repository) that are linked (which I think may be the direction of travel). We can also not fully control who has admin rights over the repository which may lead to issues down the line. In terms of developing a community/organisation distingushing the project from the work of a single research group may also be helpful. 

If we move to our own org we can control the home page, have linked repositories, set up a base site (i.e like the tidyverse) etc. It might be confusing as it would effectively be called `epinowcast/epinowcast`. The big upside for staying put is that nothing needs to be done and that it helps to be associated with a known brand (thought TBH I think the direction of benefit for this is the other way around 😉 ). 

On top of voting, any free text thoughts would be great!

**Comment by seabbs** [comment]
Note I reserved the `epinowcast` organisation: https://github.com/epinowcast

I've also reserved [epinowcast.org](http://www.epinowcast.org/) . The plan would be to build something like [tidyverse.org](https://www.tidyverse.org/) and keep packages as subdomains (again as the `tidyverse` does it). I've made a skeleton to show what this could look like.

**Comment by adrian-lison** [comment]
For me, either would be fine... The independence may be a plus indeed. What I think I am still missing is a larger vision of what helper packages etc. we would want to have in there that are specific to epinowcast. In other words, the bigger we think, the more I also lean towards having our own org for epinowcast

**Reply by seabbs** [reply]
Yeah, so this would be things like what happens if/when we want a different back-end to the stan version? What about a python implementation? Or perhaps if we want to make a case study repository, or a simplified front-end package for a single task. Depending on desire and other tooling etc there is also modularising out our tools (for example pre-processing, stan code generator, and the formula interface). 

All of that is much easier if there is more infra than a single repository. None of that may happen but I think it definitely won't if we don't shift organisation around a bit.

**Reply by adrian-lison** [reply]
Okay, that was easy. I am convinced now. +1 for own organization

**Comment by sbfnk** [comment]
I think it makes sense for projects that don't have core development limited to epiforecasts members and involve multiple repositories to move to their own organisations as we've done for the [forecast/scenarios hubs](https://github.com/covid19-forecast-hub-europe/).

If we do decide to move this out of the `epiforecasts` organisation we lose the explicit link so it would be good to mention the affiliation and acknowledge funding somewhere.

**Reply by seabbs** [reply]
>  acknowledge funding somewhere.

Yes, we can acknowledge funding for sure. Perhaps a page on epinowcast.org mentioning funding each contributor has received that is relevant to the project. We can then link to this from other places (like the package readme). I'd don't think we want to imply the project itself is fully epiforecast funded as obviously other contributors have no connection, so much of the package dev (vs the research) has happened after hours in our own time, and this might make it harder to attract project-level funding that is independent of ongoing grants.

**Comment by seabbs** [comment]
It looks like this ended up being strongly positive so will make the changes needed to move over tomorrow evening. Will aim to do this seamlessly but could cause some issues so be aware.
